schema/src/main/jade-tables/hles_owner.table.json

```
@@ -33,39 +60,39 @@
     },
     {
       "name": "od_annual_income_range_usd",
-      "datatype": "string"
+      "datatype": "integer"
     },
     {
       "name": "oc_household_person_count",
-      "datatype": "string"
+      "datatype": "integer"
     },
     {
       "name": "oc_household_adult_count",
-      "datatype": "string"
+      "datatype": "integer"
     },
     {
       "name": "oc_household_child_count",
-      "datatype": "string"
+      "datatype": "integer"
     },
     {
       "name": "ss_household_dog_count",
-      "datatype": "string"
+      "datatype": "integer"
     },
     {
       "name": "oc_primary_residence_state",
       "datatype": "string"
     },
     {
       "name": "oc_primary_residence_census_division",
-      "datatype": "string"
+      "datatype": "integer"
     },
     {
       "name": "oc_primary_residence_zip",
-      "datatype": "string"
+      "datatype": "integer"
     },
     {
       "name": "oc_primary_residence_ownership",
-      "datatype": "string"
+      "datatype": "integer"
     },
     {
       "name": "oc_primary_residence_ownership_other",
```

```suggestion
      "name": "oc_primary_residence_ownership_other_description",
```
New convention for the free-text details of "other" answers